packages/feedback/test/widget/Form.test.ts

```diff
@@ -110,6 +116,30 @@ describe('Form', () => {
     message.dispatchEvent(new KeyboardEvent('keyup'));
   });
 
+  it('submit is enabled if name is required and is not empty', () => {
```

Comment:
Ah I forgot to update the message test when changing how required works... this isn't really testing anything. Can we update these tests to check that `sendFeedback` is not being called when value is empty?